website/content/configuration/troubleshooting.md

```diff
@@ -20,7 +20,11 @@ assignment of the IP and can't connect.
 
 ## Tools
 
-### arping
+### `ping`
```

Comment:
Can you add this as a note (look for the other examples):
```
{{% notice note %}}

```

Also, I'd put this in the bottom of the file

Reply:
Also, replace OS interface with host interface